File: WebAPI/PropertyHubAPI/PropertyHubAPI/DTO/Property/PropertyWithoutOthersAttributeDto.cs
using PropertyHubAPI.DTO.Addresses;

namespace PropertyHubAPI.DTO.Property
{
    public class PropertyWithoutOthersAttributeDto
    {
        public required string MLS { get; set; }
        public required string Type { get; set; }
        public required decimal Price { get; set; }
        public required int Bedrooms { get; set; }
        public required int Bathrooms { get; set; }
        public required int Parkings { get; set; }
        public required int Size { get; set; }
        public required int YearBuilt { get; set; }
        public required decimal Tax { get; set; }
        public required AddressDto Address { get; set; }
        public required string Status { get; set; }
        public required string Description { get; set; }
    }
}

File: client/src/models/PropertyWithoutOthersAttributeDto.ts
import { AddressDto } from './AddressDto';

export interface PropertyWithoutOthersAttributeDto {
    mls: string;
    type: string;
    price: number;
    bedrooms: number;
    bathrooms: number;
    parkings: number;
    size: number;
    yearBuilt: number;
    tax: number;
    address: AddressDto;
    status: string;
    description: string;
}